webapp/src/main/java/com/box/l10n/mojito/service/assetExtraction/AssetExtractionService.java

```diff
@@ -214,7 +220,9 @@ public PollableFuture<Asset> processAsset(
         asset, createdTextUnitsResult.getUpdatedState(), currentTask);
     updatePushRun(asset, createdTextUnitsResult.getUpdatedState(), pushRunId, currentTask);
     performLeveraging(createdTextUnitsResult.getLeveragingMatches(), currentTask);
-
+    if (aiTranslationJobScheduler != null) {
```

Comment:
It might be useful to include logging to say that the job scheduler has been enabled or something (to help debug issues in the future)

Reply:
Will add a debug log where we pass to the scheduler text unit information, and another in the scheduler to indicate that translation for this text unit has been scheduled, as the scheduler will only schedule translation if an ai prompt of type translation exists in the database for the relevant mojito repository